src/Themes/VuetifyTheme/Views/Widget-VImg.liquid

```diff
@@ -9,8 +9,8 @@
   {% if alternateText %}alt="{{ alternateText }}"{% endif %}
```

Comment:
Any reason why not adding the if to these?

Reply:
Just to prevent putting an empty attribute.